### ISSUE TITLE
Card Title Wrapped during Edit in the Details View.

### ISSUE THREAD

**Issue by der-Daniel** [nextcloud/deck#865]
**Describe the bug**
This is and UI issue. When I click on a card, the details view appear on the right. When the title of the card is long, the text is wrapped with line breaks. If I click on the title to edit the text, the wrapping disappears and the text field is only a single line. This is very distracting as (1) the layout unexpectedly change and (2) editing a long text in a single line text field is a pain.

**To Reproduce**
1. Create a new card with a long title.
2. Click on the card.
3. Look at the details view on the right.
4. Click on the title in the details view.

**Expected behavior**
Keep the wrapping of the text.

**Screenshots**
Before I have clicked on the title.
![image](https://user-images.githubusercontent.com/7686866/51893083-2a222400-23a4-11e9-9dba-d0dac07307fc.png)

After I have clicked on the title.
![image](https://user-images.githubusercontent.com/7686866/51893103-38704000-23a4-11e9-9cc8-c5bf74a5a2cd.png)

**Client details:**
 - OS: Windows
 - Browser: Chrome
 - Version: 0.5.2
 - Device: desktop

**Comment by juliushaertl** [comment]
Yes, we probably should just use a contenteditable element, so we have proper support for multiple lines.

**Comment by juliushaertl** [comment]
We currently don't have editing capabilities in the sidebar in the vue rewrite, needs to be done before the release.

**Comment by juliushaertl** [comment]
This should work for both the sidebar as well as the card view.

**Comment by alexanderdd** [comment]
@juliushaertl I see you closed the bug about wrapping title in cards (https://github.com/nextcloud/deck/issues/1572) view as a duplicate of this one.
Personally, I would prefer to track wrapping in cards view vs in detail view in two different issues, because it will also be separate pull requests. But you seem to be involved in development here, so do it how you like it better.

But then can you change the title of this issue to "**wrap title in card view and details view during editing**"?

**Comment by alexanderdd** [comment]
@juliushaertl you modified the milestone various times, now it was removed. Was that intentional / is this issue lower priority now?

Just to understand.

**Comment by juliushaertl** [comment]
Not really a lower priority, but just not scheduled yet since it didn't make it into 1.2 and 1.3. We'll need to see if we can make time for it in one of the upcoming release plannings.
